package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bezier-js",
-  "version": "2.6.0",
+  "version": "2.6.1",
```

Comment:
this is managed by npm, which bumps the version _and_ automatically builds a corresponding git tag, so please don't add this manually =)

Reply:
oh okay, NP :)

Reply:
done

Reply:
Thanks a lot for fixing getLUT :)